include/miopen/miopen.h

```diff
@@ -8003,6 +8004,65 @@ MIOPEN_EXPORT miopenStatus_t miopenMultiMarginLossForward(miopenHandle_t handle,
 // CLOSEOUT LossFunction DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+// Median APIs
+/** @addtogroup median
```

Comment:
I suggest to upcase the first letter
```suggestion
/** @addtogroup Median
```

Reply:
Could you pls provide a reasonable justification for this?

Reply:
This line used by doxygen for generating document. The group name is the group name in document, a.k.a the title too. Imagine a title without Uppercase letter, you will understand what I mean.

Reply:
I see your point, but I noticed that in previous operations with single-word names, they simply used lowercase for the first letter [[ref]](https://github.com/ROCm/MIOpen/blob/31be57bb7c2135a845ab6201dde54ec28c0b5c28/include/miopen/miopen.h#L53-L74). Although this is in the title section, it's actually a function name, so i guess lowercase would be acceptable, too. 

Are there any references (such as documentation or discussions) that specify uppercase should be used for Doxygen group names? If not, I suggest we keep it lowercase for simplicity.